### ISSUE TITLE
Add line of text when there are no results in the glossary search

### ISSUE THREAD

**Issue by mcharg** [ONRR/doi-extractives-data#4393]
It feels like there should be something there to match the no results in the search.

![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/c22e3b31-f718-4598-8ce6-3799b3fb1ed9)

**Comment by brentryanjohnson** [comment]
Yes, @master12 and I are going to build that in next sprint, so the glossary text only renders when the search term is in the glossary.

**Comment by mcharg** [comment]
@brentryanjohnson It wouldn't hurt to also have the text in the glossary for when the user is searching otherwise.

**Comment by brentryanjohnson** [comment]
Oh yes, I misread your issue title. Yes, we need to add text for no result, for sure.
